docs(blog): add "Development Workflow Guide" announcement (#171)
shorebirdtech/website: src/content/blog/development-workflow.md

---
title: Development Workflow
author: felangel
description: Announcing our recommended approach for integrating Shorebird into your development workflow.
date: 2024-09-25
cover: /blog/development-workflow-cover.png
coverAlt: Development Workflow Cover Image
---

> Many of our customers have been asking for an opinionated development workflow

Reviewer: ```suggestion
Customers have asked for an opinionated development workflow
```
Fewer words. 🤷  Doesn't really matter though.
